app/models/installed_heat_source.rb

```diff
@@ -0,0 +1,18 @@
+class InstalledHeatSource
+  include Virtus.model
+
+  attribute :key, String
+  attribute :name, String
+  attribute :units, Float
+  attribute :heat_production, Float
```

Comment:
> This might seem like a stupid question: but Is the demand of a heat technology equal to the production?

Yes and no. Demand describes the total amount of energy in kWh a technology will consume in a year, production is the total amount if will produce. In the case of heat sources, I was thinking about it the same way we do with capacity, where a positive number shows consumption and a negative indicates production. I was expecting the heat sources to have a negative demand.

Maybe this is not so important: a positive "production" will certainly make more sense to a user, so perhaps we should keep it the way it is.

We do still need a way to get a production-scaled (and in the case of a weekly calculation, "cut") curve for the must-run sources. I can look into that.